_posts/2022-02-01-attribution.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Attribution"
+date:   2022-02-01 
+author: Agnese Beltramo
+category: Open Science
+tags: attribution
+---
+
+<div style="text-align: justify">
```

Comment:
Don't use styles (or any HTML at all) - this can be done at the blog level.

Reply:
Ok, I removed it from all other guidelines as well.